src/main/java/com/dsl/expressions/path/Path.java

```diff
@@ -85,8 +86,8 @@ public RelationshipPathExpression relProps(Object... o) {
   }
 
   @Override
-  public VariablePath assignTo(String name) {
-    return new VariablePath(name, this);
+  public Variable<PathExpression> assignTo(String name) {
```

Comment:
Isn't this the same idea behind the .as()? Meaning, where is the assignTo name comming from?